### PR TITLE
add 2 values to departments option list

### PR DESCRIPTION
**Pull request by kspurgin**
'naamcc' (National Afro-American Museum & Cultural Center) and 'poindexter-village' (Poindexter Village)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+o## v1.0.6
```
Comment (ray-lee) [inline CHANGELOG.md:3]: Have an extra "o" there.

**Review by ray-lee — COMMENTED** [review]
There's a typo in the changelog. Otherwise looks good!

**Comment by kspurgin** [comment]
@ray-lee - I removed extra "o". Do you need to merge it, or should I?

**Comment by ray-lee** [comment]
You could do the merge, but I need to publish the new version, so it would still be blocked on me. If I had any time I'd automate it.

**Comment by ray-lee** [comment]
1.0.6 is published.

**Pull request merged**